types/sap__approuter/sap__approuter-tests.ts

```diff
@@ -0,0 +1,139 @@
+/// <reference types="node" />
```

Comment:
I'm pretty sure it's the index that needs this reference, not the tests.

Reply:
Yes, makes more sense